apps/bttester/pkg.yml

```diff
@@ -32,10 +32,13 @@ pkg.deps:
     - "@apache-mynewt-core/sys/stats"
     - "@apache-mynewt-core/sys/shell"
     - "@apache-mynewt-nimble/nimble/host"
+    - "@apache-mynewt-nimble/nimble/host/audio"
     - "@apache-mynewt-nimble/nimble/host/util"
     - "@apache-mynewt-nimble/nimble/host/services/gap"
     - "@apache-mynewt-nimble/nimble/host/services/gatt"
     - "@apache-mynewt-nimble/nimble/host/services/dis"
+    - "@apache-mynewt-nimble/nimble/host/audio/services/pacs"
+    - "@apache-mynewt-nimble/nimble/host/audio/services/pacs/lc3"
```

Comment:
Is this package needed? You're using general API, not LC3 simplified version

Reply:
We are using LC3 e.g. with codec parameters.

Reply:
Ah, so PTS acts ok if  PACS is prepopulated. In this case, this can stay. So as far as I can see, you're only add new codec when PACS needs to be updated, right?

Reply:
Yup